.github/labeler.yml

```diff
@@ -63,3 +63,4 @@
 
 "filetype":
   - runtime/lua/vim/filetype.lua
+  - runtime/lua/vim/filetype/detect.lua
```

Comment:
Slight nitpick: could also use `runtime/lua/vim/filetype/*` if we anticipate more files in the future.

Reply:
IIRC there was a reason that filetype label is not added for `runtime/filetype.lua`, so there may also be a reason that filetype label should not added for some other files under `runtime/lua/filetype/`.